yt/python/yt/wrapper/query_commands.py

```diff
@@ -33,6 +35,9 @@ def start_query(engine, query, settings=None, files=None, stage=None, annotation
         "annotations": get_value(annotations, {}),
     }
 
+    if access_control_object is not None:
```

Comment:
there is a helper for this, called `set_param`

Reply:
ty, fixed